jsConditionsAndLoops/task1.js

```diff
@@ -0,0 +1,17 @@
+var averageGrade = 45;
+
+if(averageGrade >= 0 && averageGrade < 60){
+    console.log('Ваш результат - Незадовільно!');
+} else if(averageGrade >= 0 && averageGrade <= 70){
+    console.log('Ваш результат - Задовільно!');
+} else if(averageGrade >= 0 && averageGrade <=80){
+    console.log('Ваш результат - Добре!');
+} else if(averageGrade >= 0 && averageGrade <= 90){
```

Comment:
Цю умову взагалі можна прибрати, в першому if вже і так перевіряється, що оцінка точно більше ніж 60
```suggestion
} else if(averageGrade <= 90){
```